src/show.jl

```diff
@@ -0,0 +1,51 @@
+
+# 2-arg show, mainly for repr():
+function Base.show(io::IO, A::NamedDimsArray{L,T,N}) where {L,T,N}
+    if get(io, :typeinfo, Any) <: NamedDimsArray
+        show(io, parent(A))
+    else
+        print(io, "NamedDimsArray(")
+        show(io, parent(A))
+        print(io, ", ", N == 1 ? QuoteNode(L[1]) : L, ")")
+    end
+end
+
+# This is called by summary(), for main REPL printing:
+function Base.showarg(io::IO, A::NamedDimsArray{L,T,N}, outer) where {L,T,N}
+    print(io, "NamedDimsArray(")
+    Base.showarg(io, parent(A), false)
+    print(io, ", ", ColourString(N == 1 ? QuoteNode(L[1]) : L), ")")
+end
+
+function Base.print_matrix(io::IO, A::NamedDimsArray)
+    s1 = ColourString("↓ ", dimnames(A, 1), "  ")
+    if ndims(A)==2
+        println(io, " "^Base.Unicode.textwidth(s1), ColourString("→ ", dimnames(A, 2)))
+    end
+    ioc = IOContext(io, :displaysize => displaysize(io) .- (1, 0))
+    Base.print_matrix(ioc, parent(A), s1)
```

Comment:
```suggestion
    Base.print_matrix(ioc, parent(A), s1)
    return nothing
```

Reply:
I have to say, the robot and I disagree on which version is easier to read here...